src/Service/CognitoIdentityProvider/CHANGELOG.md

```diff
@@ -2,15 +2,17 @@
 
 ## NOT RELEASED
 
+## 0.2.1
```

Comment:
This is already in master, isnt it?

Reply:
yeah, need rebase.

in fact the issue with php8 is in httpClient (included composer2)